src/cmd_line/commandLine.ts

```diff
@@ -33,14 +33,14 @@ export class CommandLine {
       const useNeovim = configuration.enableNeovim && cmd.command && cmd.command.neovimCapable;
 
       if (useNeovim) {
-        await Neovim.command(vimState, command);
+        await vimState.nvim.run(vimState, command);
       } else {
         await cmd.execute(vimState.editor, vimState);
       }
     } catch (e) {
       if (e instanceof VimError) {
         if (e.code === ErrorCode.E492 && configuration.enableNeovim) {
-          await Neovim.command(vimState, command);
+          await vimState.nvim.run(vimState, command);
```

Comment:
@chillee, I don't quite understand this bit. If it's `E492` we want to retry running neovim again?

Reply:
When we pass the command into our own command line command/parser, there's lots of cases in which the parser will error (since it doesn't support a lot of commands).

If the parser can't parse it, then we try sending it to the neovim parser (which may support the command).

Reply:
So basically, the above code that detects if the command is neovim capable and passes it to Neovim only works for commands that we **know** are neovim capable. There's lots of commands that we don't implement, and the parser errors out in those cases.